apps/internal/jarvis/public/star.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
dont we have those svg's in s3 bucket?

Reply:
This is auto generated by the framework, I was focusing the modules and versioning task and less about the files themselves.
Let me know if you want me to remove it.